full/src/test/java/apoc/load/LoadHtmlTest.java

```diff
@@ -546,7 +546,8 @@ public static void skipIfBrowserNotPresentOrCompatible(GraphDatabaseService db,
                         resultConsumer.accept(result);
                         return true;
                     }, Duration.ofSeconds(5));
-                } catch (TransactionTerminatedException e) {
+                } catch (Exception e) {
+                    System.out.println("e = " + e);
                     // those query should generally take less than a second.
```

Comment:
I don't agree with this change. This is ignoring all exceptions here

Reply:
Yes, the catch capture any exception, 
however further down there is this construct, so it fails correctly in case of other error messages:
```
                    boolean txTimeout = ExceptionUtils.getRootCause(e).getMessage()
                            .contains("The transaction has not completed within the specified timeout");
                    if (txTimeout) {
                        return false;
                    }
                    throw e;
```